Source/WTF/wtf/RetainReleaseSwift.h

```diff
@@ -25,16 +25,4 @@
 
 #pragma once
 
-#if HAVE(SWIFT_CPP_INTEROP) // FIXME: rdar://136787800
 #include <wtf/SwiftBridging.h>
```

Comment:
Would it be straightforward to remove the `RetainReleaseSwift.h` header entirely, and switch all users to just including `SwiftBridging.h` instead? It doesn't seem to be used in the `Internal` repo so hopefully this is easy!